### ISSUE TITLE
Get Alien::GMP Added As Dependency Of Math::BigInt::GMP

### ISSUE THREAD

**Issue by wbraswell** [rsimoes/Alien-GMP#6]
I can not find an official GitHub repo for Math::BigInt::GMP, so I sent an e-mail to @plicease and jberger & all the M::BI::GMP PAUSE maintainers, wherein I politely request that we add Alien::GMP as a dependency of M::BI::GMP.

This issue is not an actual problem with Alien::GMP itself, but rather serves as a placeholder for this issue instead of having only a nebulous e-mail-only thread.  (Sorry if this breaks GitHub best practices.)

**Comment by plicease** [comment]
@wbraswell you should also open a ticket on RT https://rt.cpan.org/Public/Dist/Display.html?Name=Math-BigInt-GMP

**Comment by polettix** [comment]
FYI, I copied the ticket from @wbraswell to open a [similar ticket](https://rt.cpan.org/Ticket/Display.html?id=125018) for Math::GMP. Thanks to both!

**Comment by plicease** [comment]
@polettix I don't think it would be hard to create a PR to add this support `Math::GMP` is already using `[MakeMaker::Awesome]`:

https://github.com/turnstep/Math-GMP/blob/c194194a45f0df869c2aeeaf50d4dba7e36df49c/dist.ini#L17

and there is a recipe of sorts in `Alien::Build::Manual::AlienUser`:

https://metacpan.org/pod/distribution/Alien-Build/lib/Alien/Build/Manual/AlienUser.pod#Dist::Zilla

cc: @shlomif

**Comment by wbraswell** [comment]
Howdy Y'all,

Sorry for the delay on this, we got the "go ahead" response from Jonathan Leto a while back (copied below).

@plicease  Since we did not receive any specific instructions on how to move forward with this, and there does not seem to be a GitHub repo for Math::BigInt::GMP, how do you suggest we proceed?

Thanks,
~ Will

On 12/27/2017 at 10:16 PM, "Jonathan Duke Leto" <jonathan@leto.net> wrote:
Howdy,

In general I am supportive of the Alien::* namespace, this sounds like
a good idea.

+1 from me

Duke

On Wed, Dec 13, 2017 at 1:10 AM, William N. Braswell, Jr.

**Comment by plicease** [comment]
@wbraswell Check if there is a git repo, otherwise we can provide a diff for Duke to apply against the latest version.  Looks like he is using EUMM so this is the recommended recipe for that:

https://metacpan.org/pod/distribution/Alien-Build/lib/Alien/Build/Manual/AlienUser.pod#ExtUtils::MakeMaker

There is an old repo which appears to be what the previous maintainer was using:

https://github.com/rafl/Math-BigInt-GMP
